### PR TITLE
Add a clarification about IP Address for Microsoft content

### PR DESCRIPTION
**Pull request by narkissit**
## Why

- Closes #[Issue Number]

### CODE REVIEW

File: windows/deployment/do/waas-delivery-optimization-faq.yml [hunk 1]
```diff
@@ -57,6 +57,11 @@ sections:
 
           For more information, see [Endpoints for Delivery Optimization and Microsoft Connected Cache](../do/delivery-optimization-endpoints.md) for a list of all content endpoints needed.
 
+     - question: My firewall requires IP Address and cannot process FQDN's, how do I configure it to download content with Delivery Optimization?
+        answer: | 
+          Microsoft content such as Windows Updates are hosted and delivered globally via Content Delivery Networks (CDNs) as well as Microsoft's Connected Cache (MCC) servers which are hosted within Internet Service Provider (ISP) networks. 
+          The network of CDNs and MCCs allows Microsoft to reach the scale required to meet the demand of the Windows user base. Given this delivery infrastructure changes dynamically, providing an exhaustive list of IPs and keeping it up to date is not feasible.
```
Comment (JohanFreelancer9) [inline windows/deployment/do/waas-delivery-optimization-faq.yml:63]:
```suggestion
     - question: My firewall requires an IP Address and cannot process FQDNs, how do I configure it to download content with Delivery Optimization?
        answer: | 
          Microsoft content such as Windows Updates are hosted and delivered globally via Content Delivery Networks (CDNs) as well as Microsoft's Connected Cache (MCC) servers which are hosted within Internet Service Provider (ISP) networks. 
          The network of CDNs and MCCs allows Microsoft to reach the scale required to meet the demand of the Windows user base. Given this delivery infrastructure changes dynamically, providing an exhaustive list of IPs and keeping it up to date is not feasible. 
```

File: windows/deployment/do/waas-delivery-optimization-faq.yml [hunk 2]
```diff
@@ -57,6 +57,11 @@ sections:
 
           For more information, see [Endpoints for Delivery Optimization and Microsoft Connected Cache](../do/delivery-optimization-endpoints.md) for a list of all content endpoints needed.
 
+     - question: My firewall requires IP Address and cannot process FQDN's, how do I configure it to download content with Delivery Optimization?
```
Comment (JohanFreelancer9) [inline windows/deployment/do/waas-delivery-optimization-faq.yml:60]:
```suggestion
     - question: My firewall requires an IP Address and cannot process FQDNs, how do I configure it to download content with Delivery Optimization?
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @narkissit, kindly commit the minor suggested changes. Many thanks.

**Comment by cmknox** [comment]
@JohanFreelancer9 can you take a look again, I made the change you requested. Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mestew** [comment]
I pulled Narkis's branch into the private to work on it. Closing this PR.

**Pull request closed without merging**